src/Aks/Aks.Test/ScenarioTests/KubernetesTests.ps1

```diff
@@ -1161,4 +1161,42 @@ function Test-PodSubnetID {
     finally {
         Remove-AzResourceGroup -Name $resourceGroupName -Force
     }
+}
+
+function Test-EnableOidcIssuer {
+    # Setup
+    $resourceGroupName = Get-RandomResourceGroupName
+    $kubeClusterName1 = Get-RandomClusterName
+    $kubeClusterName2 = Get-RandomClusterName
+    $location = 'eastus'
+    $nodeVmSize = "Standard_D2_v2"
+
+    try {
+        New-AzResourceGroup -Name $resourceGroupName -Location $location
+        
+        New-AzAksCluster -ResourceGroupName $resourceGroupName -Name $kubeClusterName1 -NodeVmSize $nodeVmSize -NodeCount 1 -EnableOidcIssuer
+        $cluster1 = Get-AzAksCluster -ResourceGroupName $resourceGroupName -Name $kubeClusterName1
+        Assert-True {$cluster1.OidcIssuerProfile.Enabled}
+        Assert-True {$cluster1.OidcIssuerProfile.IssuerURL.StartsWith("https://eastus.oic.prod-aks.azure.com")}
+        
+        # set EnableOidcIssuer=false should not work.
+        Set-AzAksCluster -ResourceGroupName $resourceGroupName -Name $kubeClusterName1 -EnableOidcIssuer:$false
+        $cluster1 = Get-AzAksCluster -ResourceGroupName $resourceGroupName -Name $kubeClusterName1
+        Assert-True {$cluster1.OidcIssuerProfile.Enabled}
+        Assert-True {$cluster1.OidcIssuerProfile.IssuerURL.StartsWith("https://eastus.oic.prod-aks.azure.com")}
```

Comment:
The ability to disable OIDC in the `Set-AzAksCluster` command is managed by the server-side. There is no need to record test for this feature.

Reply:
Removed.